docs/devsguide/new_dev_guide.rst

```diff
@@ -1,133 +1,213 @@
 *********************
 New Developer's Guide
 *********************
-=================
-Setting up Github
-=================
+
+==================
+Setting up Github:
+==================
 Before you start using GitHub, you have to make Git available on your computer.
-Even if it’s already installed, it’s probably a good idea to update to the 
-latest version. You can either install it as a package or via another installer, 
+Even if it’s already installed, it’s probably a good idea to update to the
+latest version. You can either install it as a package or via another installer
 or download the source code and compile it yourself.
 
-For download of Git visit `here <http://git-scm.com/downloads>`__
+To download Git visit 
+https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
 
 -----------------
 Installing on Mac
 -----------------
-On Mavericks (10.9) or above you can do this simply by trying to run git from 
-the Terminal the very first time. If you don’t have it installed already, it 
-will prompt you to install it. You might be prompted to install command line 
-developer tools if Xcode isn’t installed on your computer. If you want a more up 
-to date version, you can also install it via a binary installer. An OSX Git 
-installer is maintained and available for download at the `Git website 
-<http://git-scm.com/download/mac.>`__.
+On Mavericks (10.9) or above you can do this simply by trying to run git from
+the Terminal the very first time. If you don’t have it installed already, it
+will prompt you to install it. You might be prompted to install command line
+developer tools if Xcode isn’t installed on your computer. 
+
+If you want a more up to date version, you can also install it via a binary 
+installer. An OSX Git installer is maintained and available for download at 
+the http://git-scm.com/download/mac. This method will require Homebrew, which
+may not already be on your device.
+
+
+To Install Homebrew
+```````````````````
+Run the commands ::
+
+	$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+	$ brew doctor
+	$ brew tap brewsci/science
 
 ---------------------
 Installing on Windows
```

Comment:
Has anyone actually build PyNE on Windows recently (ever?)

Reply:
If yes, then super!!! if not, then I wonder if we should remove this section?

Reply:
I have not built it on Windows, but on Mac OS I found this step necessary.

Reply:
Roger that.  Maybe we'll make an issue for someone to try it in a Windows conda environment (if we can find one).